recipes/pyneval/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pyneval" %}
+{% set version = "0.0.0" %}
```

Comment:
It looks like the latest version is `0.0.2`. You can keep the current version and then put in PR for the update once the feedstock is created

Reply:
Thank you, sir. I'll fix these problems and try it on a private channel first. Thank you for your recommendations.

Reply:
You're welcome do that, but the changes should be pretty small and you can just commit to the branch associated with this PR. The recipe only needs a few changes and then I'd be happy to merge it.